project/plugins.sbt

```diff
@@ -1,7 +1,7 @@
 addSbtPlugin("org.scalameta"      % "sbt-scalafmt"             % "2.4.6")
 addSbtPlugin("io.crashbox"        % "sbt-gpg"                  % "0.2.1")
-addSbtPlugin("com.github.sbt"     % "sbt-release"              % "1.1.0")
-addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"             % "3.9.12")
-addSbtPlugin("de.heikoseeberger"  % "sbt-header"               % "5.7.0")
-addSbtPlugin("org.scala-js"       % "sbt-scalajs"              % "1.10.0")
-addSbtPlugin("org.portable-scala" % "sbt-scalajs-crossproject" % "1.2.0")
+addSbtPlugin("com.github.sbt"     % "sbt-release"              % "1.4.0")
+addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"             % "3.9.21")
```

Comment:
steward says 3.10.0

Reply:
when I visited https://github.com/xerial/sbt-sonatype I looked at the "maven badge" instead of Releases / "latest".